Src/PerformanceCollector/Shared/Implementation/QuickPulse/Process/IQuickPulsePerfLib.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.QuickPulse
+{
+    using Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.QuickPulse.PerfLib;
+
+    /// <summary>
+    /// Interface for the <c>PerfLib</c>.
+    /// </summary>
+    internal interface IQuickPulsePerfLib
```

Comment:
Instead of custom interfaces - I'd suggest to change the design:
1. Use the same method to read counters for both - standard counters and processes
2. Support '*' instance name in that interface
3. Extract top 5 and total on "query" side when creating the message to quick pulse.

This implementation looks like a band aid.